src/serac/physics/materials/material_verification_tools.hpp

```diff
@@ -80,6 +80,26 @@ auto uniaxial_stress_test(double t_max, size_t num_steps, const MaterialType mat
   return output_history;
 }
 
+/**
+ * @brief Drive a rate-dependent material model thorugh a uniaxial tension experiment
+ *
+ * Drives material model through specified axial displacement gradient history.
+ * The time elaspses from 0 up to t_max.
+ * Currently only implemented for isotropic materials (or orthotropic materials with the
+ * principal axes aligned with the coordinate directions).
```

Comment:
```suggestion
 * principal axis aligned with the coordinate directions).
```

Reply:
This one isn't a typo, it's "axes" as in the plural of "axis".

Reply:
axiseseses